bindings/node.js/binding.gyp

```diff
@@ -0,0 +1,51 @@
+{
+  'targets': [
+    {
+      'target_name': 'blst-node-binding',
+      'sources': [
+        'blst_wrap.cpp',
+        '../libblst.a'
+      ],
+      # Is 'libblst.a' a library or a source?
+      'libraries': [],
+      'include_dirs': [
+        '../',
+        # '<!(node -e \"require(\'nan\')\")'
+      ],
+      
+      'cflags': [
+        '-shared',
+        '-fPIC',
+        '-std=c++11',
+        '-DBUILDING_NODE_EXTENSION',
+      ],
+      # node-gyp has both cflags_cc and cflags. Not sure which one to use
+      'cflags_cc': [
+        '-O',
+        '-Wall',
+        '-Wno-unused-result',
+        '-Wl,-Bsymbolic',
+      ],
+      # Also has 'defines' for flags or ENV variables?
+      'defines': [],
+
+      'conditions': [
+        ['OS=="mac"', {
+          # Exclusion list (!): removes items from the main 'cflags' array
+          'cflags!': [
+            '-shared',
+          ],
+          'cflags_cc!': [
+            '-Wl,-Bsymbolic',
+          ],
+          'cflags': [
+            '-bundle',
+            '-bundle_loader $NODE'
```

Comment:
As for `-shared`, `-bundle`, `-bundle_loader` flags, my understanding is that node-gyp should take care of those. At least the whole idea behind it is to liberate you from thinking about such platform specifics... So just remove them as starting point. BTW, `-Wl,-Bsymbolic` is effectively optional. Everything should work without it, but it works slightly better with. In other words it can also be omitted as starting point.